ConfigSpace/hyperparameters.pyx

```diff
@@ -877,7 +877,7 @@ cdef class UniformIntegerHyperparameter(IntegerHyperparameter):
         if self.log:
             repr_str.write(", on log-scale")
         if self.q is not None:
-            repr_str.write(", Q: %s" % repr(np.int(self.q)))
+            repr_str.write(", Q: %s" % repr(np.int64(self.q)))
```

Comment:
Do we have a test for this change?

Reply:
Thanks, I just added a test for this.

Reply:
Ha, I'm wondering why we cast here in the first place. We don't cast for upper and lower, so this seems superfluous, what do you think?